packages/cna-template/template/nuxt/nuxt.config.js

```diff
@@ -105,6 +105,10 @@
     // https://go.nuxtjs.dev/vuetify
     '@nuxtjs/vuetify',
     <%_ } _%>
+    <%_ if (composition) { _%>
+    // https://composition-api.nuxtjs.org/
+    '@nuxtjs/composition-api',
```

Comment:
For testing in jest, let's add moduleNameMapper in jest config

```js
moduleNameMapper: {
  '@nuxtjs/composition-api': '@nuxtjs/composition-api/lib/entrypoint.js',
  // alternatively, depending on your node version
  // '@nuxtjs/composition-api': '@nuxtjs/composition-api/entrypoint',
},
```

Reply:
@clarkdo Good point 👍  
Chech this commit please.
https://github.com/nuxt/create-nuxt-app/pull/759/commits/4bfcdf45b34790a143eca422b6284ef45c87433d